src/resources/locales/en.json

```diff
@@ -211,7 +211,7 @@
         "annotations": "Annotations"
       },
       "svg": {
-        "contentTable": "Content Table",
+        "contentTable": "Navigation Table",
```

Comment:
A couple of questions:
* Where is this label used?
* Should this be "table of contents"? (I'm not sure, depends where the label is used)

Reply:
cf Fernando meeting,
@llemeurfr ?

Reply:
New issue to be addressed separately https://github.com/readium/readium-desktop/issues/396